db/fixtures/miq_product_features.yml

```diff
@@ -3729,36 +3729,15 @@
   :feature_type: node
   :identifier: container_image
   :children:
-  - :name: View
-    :description: View Container Images
-    :feature_type: view
-    :identifier: container_image_view
-    :children:
-    - :name: List
-      :description: Display Lists of Container Images
-      :feature_type: view
-      :identifier: container_image_show_list
-    - :name: Show
-      :description: Display Individual Container images
-      :feature_type: view
-      :identifier: container_image_show
-  - :name: Modify
-    :description: Modify Container Images
-    :feature_type: admin
-    :identifier: container_image_admin
+  - :name: Operate
+    :description: Perform Operations on Container Images
+    :feature_type: control
+    :identifier: container_image_control
     :children:
-    - :name: Remove
-      :description: Remove Container Images from the VMDB
-      :feature_type: admin
-      :identifier: container_image_delete
-    - :name: Edit
-      :description: Edit a Container Image
-      :feature_type: admin
-      :identifier: container_image_edit
-    - :name: Add
-      :description: Add a Container Image
-      :feature_type: admin
-      :identifier: container_image_new
+    - :name: Analysis
+      :description: Perform SmartState Analysis for Images
+      :feature_type: control
+      :identifier: image_scan
```

Comment:
@oschreib another feature with same id("image_scan") exists in miq_product_features.yml for cloud images, can you rename this button feature id to be unique